rules/web/web_cve_2021_44228_log4j_fields.yml

```diff
@@ -111,7 +114,7 @@ detection:
          - '${${lower:j}ndi:${lower:l}${lower:d}a${lower:p}://'
          - '${${upper:j}ndi:${upper:l}${upper:d}a${lower:p}://'
          - '${${::-j}${::-n}${::-d}${::-i}:'
-   condition: selection
+   condition: selection1 OR selection2 OR selection3 OR selection4
```

Comment:
to fix to error
`condition: selection1 or selection2 or selection3 or selection4`
OR
`condition:  1 of selection*`

Reply:
great suggestion, thanks